cpp/src/arrow/ipc/metadata_internal.h

```diff
@@ -58,6 +58,9 @@ class DictionaryMemo;
 
 namespace internal {
 
+// This 0xFFFFFFFF value is the first 4 bytes of a valid IPC message
+constexpr int32_t kIpcContinuationToken = -1;
```

Comment:
for some reason i thought there were linkage issues in C++11 using constexpr in a header.